app/views/accounts/actions/_edit_action_button.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= link_to path do %>
-  <span class='hide'>
+  <span class='display-none'>
     <%= name %>
   </span>
```

Comment:
Not relevant for the purpose of the pull request, but I'm curious what purpose these hidden action elements are serving. As far as I can tell, they're never toggled visible.

Maybe they were expected to be used as an accessible text to contextualize the "Edit" label ("Password Edit", etc)? But `display: none` are ignored by assistive technology, so this would not be happening correctly if that's the intention.

Reply:
Ya the screen reader explanation makes the most sense to me, I thought we used `hide` because it *didn't* use `display: none`, per previous convo: https://github.com/18F/identity-idp/pull/4418#discussion_r527016377

Should we keep `hide` here or switch to some sort of screen-reader-only style?

Reply:
Oh! Yes, that makes total sense. I think I was confused because `.sm-hide`, `.md-hide`, etc. classes will apply `display: none;`, and assumed `.hide` behaved similarly, when in reality in acts as a visually-hidden effect. I'll definitely need to revise this. I expect we can use the `.usa-sr-only` helper in place of `.hide`, since it serves a similar purpose.

Reply:
That being said, as I look at a few of the examples where I've changed from `hide` to `display-none`, it feels like the intention was to hide entirely, and not to explicitly make the text available for assistive tech 🤔  It could be worth the effort to consider these on a more case-by-case basis, since it may be problematic that we leave them exposed to the accessibility tree, even if that would be the direct port equivalent of what we have today.

Reply:
Summarized in self-review at https://github.com/18F/identity-idp/pull/4638#pullrequestreview-582370931 and changed some `display-none` to `usa-sr-only` in 4500c38.